### PR TITLE
tsuba: complain instead of fail when combine fails

### PR DESCRIPTION
**Pull request by tylershunt**
For LDBC 100 this arrow call fails with:
"Invalid: Must pass at least one array"

For one property; It's confusing because it does have arrays and they
appear to contain a reasonable amount of data. I suspect its either a
problem with the way we handle partial table reads.

This fix allows it to continue LDBC 100 then OOMs; I seem to recall that
we depend on the arrays being merged, but I thought I would put this
up to try and be helpful.

Signed-off-by: Tyler Hunt <thunt@katanagraph.com>

### CODE REVIEW

File: libtsuba/src/RDG.cpp [hunk 1]
```diff
@@ -372,12 +376,13 @@ DoLoadPartialTable(const std::string& expected_name,
 
   auto combine_result = out->CombineChunks(arrow::default_memory_pool());
   if (!combine_result.ok()) {
-    GALOIS_LOG_DEBUG("arrow error: {}", combine_result.status());
-    return tsuba::ErrorCode::ArrowError;
+    GALOIS_LOG_DEBUG("{}, arrow error: {}", tsuba::Comm()->ID,
```
Comment (roshandathathri) [inline libtsuba/src/RDG.cpp:379]: Better to warn as this is not returned as an error anymore. Silent errors during release builds will be confusing.

**Review by roshandathathri — APPROVED** [review]
LGTM

File: libtsuba/src/RDG.cpp (hunk 1 above)
Comment (roshandathathri) [reply libtsuba/src/RDG.cpp:379]: Also, it would useful to report the name and/or type of the property for which this error occurs.

Comment (ddn0) [reply libtsuba/src/RDG.cpp:379]: The analytics applications do depend on chunks being combined into one. My preference would be to keep that invariant for all users of properties (querying, analytics) and perhaps figure out the root cause of the issue with LDBC 100.

If we do go forward with this change (perhaps to ease deployment during testing), can you file a ticket to revisit this change?

I think reporting the column in question in the debug or warn message would be an improvement regardless.

**Comment by tylershunt** [comment]
Will apply @roshandathathri 's suggestions. I'm not sure if this violates some assumption elsewhere so I think we should hold off on merging until were confident it works (i.e. run something that has this behavior and also runs to completion).
cc. @ddn0

**Comment by tylershunt** [comment]
Closing this because I think we agree its the wrong thing.

**Pull request closed without merging**